### DISCUSSION TITLE
How can I make fullscreen for forkphorus possible?

### DISCUSSION THREAD

**Discussion by NoobTracker** [ScratchAddons/ScratchAddons#215] [Help]
My addon that adds fork and stuff to the projects currently doesn't support fork's fullscreen mode, you just end up with some pretty weird stuff.

Why is that a problem? The normal embed iframe supports it.

**Comment by WorldLanguages** [comment]
"iframe allow fullscreen" on stackoverflow.

**Comment by NoobTracker** [comment]
Ah let me try that.

**Comment by NoobTracker** [comment]
Okay, fullscreen is now working. #216 is still a thing, now I found a second project where that's the case.
